build.sbt

```diff
@@ -1,8 +1,8 @@
-lazy val scalaVersionsJVM = Seq("3.0.0", "2.13.6", "2.12.13", "2.11.12", "2.10.7")
-lazy val scalaVersionsSN  = Seq("2.13.6", "2.12.13", "2.11.12")
-lazy val scalaVersionsJS  = Seq("3.0.0", "2.13.6", "2.12.13", "2.11.12")
+lazy val scalaVersionsJVM = Seq("3.3.4", "2.13.15", "2.12.20", "2.11.12", "2.10.7")
```

Comment:
I wasn't following Scala 3.x developments, so I have to ask - we don't need to cross-build against each minor release anymore? (i.e. 3.2, 3.1, 3.0 are not needed here?)

Reply:
Thanks, fixed for 3.2 and 3.1.
I have not added Scala 3.0, because shared sources are not compatible between  3.0 and 2.13.15 due to different Tasty reader versions. This was the initial reason for Scala 3 version update.

To give more details, if we try to build for 3.0, there will be an error:    
`(class scala.tools.tasty.UnpickleException/TASTy signature has wrong version.
 expected: {majorVersion: 28, minorVersion: 0}
 found   : {majorVersion: 28, minorVersion: 1}`